modules/cudalegacy/src/calib3d.cpp

```diff
@@ -283,7 +283,7 @@ void cv::cuda::solvePnPRansac(const Mat& object, const Mat& image, const Mat& ca
             p_transf.z = rot[6] * p.x + rot[7] * p.y + rot[8] * p.z + transl[2];
             p_proj.x = p_transf.x / p_transf.z;
             p_proj.y = p_transf.y / p_transf.z;
-            if (norm(p_proj - image_normalized.at<Point2f>(0, i)) < max_dist)
+            if (norm(p_proj - image_normalized.at<Point2f>(i, 0)) < max_dist)
```

Comment:
BTW, `Mat::at(i)` should be correct too (for both `1 x n` / `n x 1` layouts) - [method](https://github.com/opencv/opencv/blame/4.5.1/modules/core/include/opencv2/core/mat.hpp#L1892-L1895) has complex [implementation](https://github.com/opencv/opencv/blame/4.5.1/modules/core/include/opencv2/core/mat.inl.hpp#L917-L920)

Reply:
Thank you.
Let me modify it to `Mat::at(i)`